main.tf

```
@@ -81,3 +81,21 @@ resource "google_compute_route" "route" {
     "google_compute_subnetwork.subnetwork",
   ]
 }
+
+resource "null_resource" "delete_default_internet_gateway_routes" {
+  count = "${var.delete_default_internet_gateway_routes != "" ? 1 : 0}"
```

`var.delete_default_internet_gateway_routes` should be treated as a boolean.

```suggestion
  count = "${var.delete_default_internet_gateway_routes ? 1 : 0}"
```